server/controllers/finance/fiscal.js

```diff
@@ -643,3 +644,28 @@ function getPeriodCurrent(date) {
 
   return db.exec(sql, [date, date]);
 }
+
+/**
+ * @method getPeriodFiscalYear
```

Comment:
I think a very similar function was also introduced in @DedrickEnc 's pull request (https://github.com/IMA-WorldHealth/bhima-2.X/pull/1813).

These should eventually be combined to make sure we don't have duplicated code - can you make an issue to track this? 
* Period select component should be combined 
* Fiscal year period utilities on the server should be combined